ledger-service/http-json/src/main/scala/com/digitalasset/http/CommandService.scala

```diff
@@ -248,11 +256,16 @@ class CommandService(
 }
 
 object CommandService {
-  final case class Error(id: Symbol, message: String)
+  sealed trait Error
+  final case class FutureFailedButAlreadyLogged(err: Throwable) extends Error
+  final case class CommandError(id: Symbol, message: String) extends Error
```

Comment:
I don’t think the separate types help here. Can we just have a single type and if needed make the id optional?

Reply:
Well that would be an option too I guess :thinking:

Reply:
adressed with 9fcdba8